tools/ci/report_coverage

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-mkdir -p coverage
-gsutil rsync gs://codeclimate-aggregation/${CIRCLE_WORKFLOW_ID} coverage
-cc-test-reporter sum-coverage coverage/codeclimate.*.json
-cc-test-reporter upload-coverage
+if [ -z "$GCLOUD_SERVICE_KEY" ]
+then
+  echo "Skipping report generation because no gcloud service key is set"
```

Comment:
Could we add an `exit 0` line after this? Potential source of bugs if someone appends to this script and it would get executed